jsk_topic_tools/src/lightweight_throttle_nodelet.cpp

```diff
@@ -95,12 +106,13 @@ namespace jsk_topic_tools
       pub_ = pnh_.advertise(opts);
     }
     ros::Time now = ros::Time::now();
-    if ((now - latest_stamp_).toSec() > 1 / update_rate_) {
+    if (update_rate_ > 0.0 && (now - latest_stamp_).toSec() > 1.0 / update_rate_) {
```

Comment:
Why we must convert type implicitly?
For me `update_rate_` is double and `0.0` is also double, so it looks fine.